rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -370,6 +370,10 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
         amount = asset_normalized_value(amount=raw_amount, asset=self.vcow_token)
         airdrop_identifier: Literal['cow_mainnet', 'cow_gnosis'] = 'cow_mainnet' if self.evm_inquirer.chain_id == ChainID.ETHEREUM else 'cow_gnosis'  # noqa: E501
         for event in context.decoded_events:
+            if event.event_type == HistoryEventType.SPEND and event.counterparty != 'gas':
```

Comment:
Is rare that you need to compare against strings in our code. There is almost always a constant 

```suggestion
            if event.event_type == HistoryEventType.SPEND and event.counterparty != CPT_GAS:
```